Sources/Datadog/RUMMonitor.swift

```diff
@@ -629,12 +635,11 @@ public class RUMMonitor: DDRUMMonitor, RUMCommandSubscriber {
         // update the core context with rum context
         core.set(feature: "rum", attributes: {
             self.queue.sync {
-                let context = self.applicationScope.sessionScope?.viewScopes.last?.context ??
-                                self.applicationScope.sessionScope?.context ??
-                                self.applicationScope.context
```

Comment:
I think this odd formatting is more common for `??`, but if linter is not complaining I think it's fine

Reply:
TBH I think the linter is what changed it.